server/routes/config/curfew.js

```diff
@@ -64,6 +64,14 @@ module.exports = {
       change: '/hdc/review/licenceDetails/',
     },
   },
+  approvedPremisesChoice: {
+    pageDataMap: ['licence'],
+    nextPath: {
+      discriminator: 'decision',
+      ApprovedPremises: '/hdc/curfew/approvedPremisesAddress/',
+      OptOut: '/hdc/taskList/',
+    },
+  },
```

Comment:
config for the standard route handler - this says where to go next based on the user's answer to the question